### ISSUE TITLE
I18n Guide error 

### ISSUE THREAD

**Issue by tatsujb** [angular/angular#24473]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

https://angular.io/guide/i18n#the-app-and-its-translation-file

right here in the example `src/locale/messages.fr.xlf` there's a duplicated extra closing tag line 28. 🌹

**Comment by tatsujb** [comment]
I'm short on time but I'll try. there's a first to everything I guess.

**Comment by brandonroberts** [comment]
I can assist as needed. Here is the file https://github.com/angular/angular/blob/master/aio/content/examples/i18n/doc-files/messages.fr.xlf.html#L44

1. Click on the pencil and edit the file to remove that line
2. Add a commit message like `docs: remove extra line in translation file`
3. Click `Create a new branch for this commit and start a pull request`
4. Click Propose File Change

Once it passes all the necessary checks, it should be good to go.

**Comment by tatsujb** [comment]
thanks alot!!!
the process was different. 
Once I clicked on the pen there was an automatic fork into my github of angular. but from where I was in that editing mode I didn't have the button you mentioned nor the right to do anything.

I had to go to my profile, 
to the new forked repo, 
to the specified file,
edit,
and then `Create a new branch for this commit and start a pull request` did indeed apear next to commit at the bottom of the edit view.

from there I finalized the PR fix and also merged (not sure if that last bit was necessary)
